.env

```diff
@@ -1 +1,2 @@
 REACT_APP_CONTROLLER_URL=wss://jimm.jujucharms.com/api
+SASS_PATH=node_modules:src
```

Comment:
Can we have comments in this file? It would be nice to add a comment as to where these are used.